nncf/quantization/algorithms/weight_compression/awq.py

```diff
@@ -181,11 +180,16 @@ def apply(
             stats = self._activations[k]
             X = fns.stack([fns.mean(stat, axis=0) for stat in stats])
             X = fns.transpose(X)
-            if X.shape[1] > self._subset_size:
-                X = X[:, : self._subset_size]
 
             s = fns.max(fns.abs(X), axis=1)
 
+            if X.shape[1] > self._subset_size:
+                # X = X[:, : self._subset_size]
```

Comment:
Don't forget to deal with the commented line

Reply:
Done